libs/src/utils.js

```diff
@@ -1,8 +1,10 @@
 const bs58 = require('bs58')
 const Web3 = require('./web3')
 const axios = require('axios')
-const MultiProvider = require('./utils/multiProvider')
-const uuid = require('./utils/uuid')
+
+const UtilsDir = require('./utils/')
```

Comment:
can we explicitly make it `require('./utils/index.js')`? i know that's how it behaves but just to reduce potential future confusion

Reply:
yeah i thought the convention was to do it this way, i prefer the exlpicit way as well, will change